### DISCUSSION TITLE
HTML version of the User Guide?

### DISCUSSION THREAD

**Discussion by trusktr** [nasa/openmct#4342] [Community Q&A]
That would be more convenient (to me) than the [PDF version](https://nasa.github.io/openmct/static/files/Open_MCT_Users_Guide.pdf) that the [Documentation](https://nasa.github.io/openmct/documentation/) page links to.

Where is the User Guide source code/content?

**Comment by charlesh88** [comment]
This is authored and managed in Adobe InDesign. What advantages do you think an HTML implementation would have over a PDF in this case?

**Reply by trusktr** [reply]
It would be easily navigable in a web browser (meaningful search engine results, navigating to anchors, find in page, etc)

**Reply by trusktr** [reply]
Another advantage is that if the source code for that is here on GitHub, people can contribute to it easily.

**Comment by akhenry** [comment]
FWIW I had a situation today where it would have been useful to be able to provide a link to a specific section of the user's guide.

**Reply by trusktr** [reply]
This is what I meant with "navigating to anchors" in my comment to @charlesh88. Indeed useful.

Anchor links also serve as breadcrumbs in the browser, such that clicking links to anchors allows the back button to navigate to previous locations in reverse order.

More reasons:

Maybe not a big one in this case, but more accessibility: browsers can toggle "reading mode", to show an alternate form of a page suited for plain reading no matter how the original layout styling works. Not sure I'd use this myself with OpenMCT docs, but it's there.

If docs source code is on GitHub, we could, publish older versions of docs for older versions of OpenMCT from previous commits. Users could pick the version from a dropdown.

**Comment by charlesh88** [comment]
We are actively looking into this and agree this is a superior solution. I need to try out the "publish to web" feature from Adobe InDesign (which is where the guide is edited and managed now) or find a solid alternative.
